cmd/configure_raid.go

```
@@ -66,13 +62,5 @@ func callLinuxSWRaid(array RaidArray) {
 		"--force", "--run", "--level", array.Level, "--raid-devices",
 		strconv.Itoa(len(array.Devices))}
 	cmd_args = append(cmd_args, array.Devices...)
-	cmd := exec.Command("mdadm", cmd_args...)
-	
-	logger.Debugw("running mdadm", "cmd", cmd)
-
-	if out, err := cmd.CombinedOutput(); err != nil {
-		logger.Debugf("%s\n", out)
-		logger.Fatalw("Failed to run mdadm",
-			"array", array, "cmd", cmd, "err", err, "stderr", out)
-	}
+	_, _ = callCommand("mdadm", cmd_args...)
```

this is tickling the spidey senses, why are you ignoring err return?